src/main/scala/org/bitcoins/core/script/locktime/LockTimeInterpreter.scala

```diff
@@ -158,18 +159,18 @@ trait LockTimeInterpreter extends BitcoinSLogger {
     // unless the type of nSequenceMasked being tested is the same as
     // the nSequenceMasked in the transaction.
     if (!(
-      (txToSequenceMasked < ScriptNumber(TransactionConstants.sequenceLockTimeTypeFlag) &&
-        nSequenceMasked < ScriptNumber(TransactionConstants.sequenceLockTimeTypeFlag)) ||
-        (txToSequenceMasked >= ScriptNumber(TransactionConstants.sequenceLockTimeTypeFlag) &&
-          nSequenceMasked >= ScriptNumber(TransactionConstants.sequenceLockTimeTypeFlag))
+      (txToSequenceMasked.underlying < TransactionConstants.sequenceLockTimeTypeFlag &&
+        nSequenceMasked.underlying < TransactionConstants.sequenceLockTimeTypeFlag) ||
```

Comment:
Implement `<` and `>=` on `ScriptNumber` to take in a `Int64` like this https://github.com/bitcoin/bitcoin/blob/d612837814020ae832499d18e6ee5eb919a87907/src/script/script.h#L244